### PR TITLE
add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by javeme**
Change-Id: I49958cd08c2c88b4a5638fc9f609be3702a7a570

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,150 @@
+# How to Contribute to HugeGraph
+
+As an open source project, HugeGraph is looking forward to be contributed from everyone, and we are also grateful to all of the contributors.
+
+We will describe the contribution process of HugeGraph as follows:
+
+## 1. Preparation
+
+Before we submit the code, we need to do some preparation:
```
Comment (Linary) [inline CONTRIBUTING.md]: maybe can remove the first 'we'

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
+# How to Contribute to HugeGraph
+
+Thanks for taking the time to contribute! As an open source project, HugeGraph is looking forward to be contributed from everyone, and we are also grateful to all of the contributors.
+
+The following is a guide for the contribution process of HugeGraph:
```
Comment (zhoney) [inline CONTRIBUTING.md]: The following is a contribution guide for HugeGraph

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+## 1. Preparation
+
+We can contribute by reporting issue or submitting code patch or any other feedback.
```
Comment (zhoney) [inline CONTRIBUTING.md]: We can contribute by reporting issues, submitting code patches or any other feedback.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+   # set name and email to push code to github
+   git config user.name "{full-name}" # like "Jermy Li"
+   git config user.email "{email-address-of-github}" # like "jermy@gmail.com"
+   ```
+
+5. Sign the HugeGraph CLA: [https://cla-assistant.io/hugegraph/hugegraph](https://cla-assistant.io/hugegraph/hugegraph)
+
+## 2. Create an Issue on GitHub
+
+If you encounter bugs or have any questions, please go to [GitHub Issues](https://github.com/hugegraph/hugegraph/issues) to report it and feel free to [create an issue](https://github.com/hugegraph/hugegraph/issues/new).
```
Comment (zhoney) [inline CONTRIBUTING.md]: report it -> report them

**Pull request merged**